savoirfairelinux/sous-chef
# Fix issue #459 double sidebar.

## Fixes #459 
### Changes proposed in this pull request:

* Fronted changes to made only one sidebar visible when resizing the window.

### Status


### How to verify this change

* Navigate to the main menu in a browser window more than 1200px wide.
* Observe that the sidebar is open.
* Reduce the width of the window to less than 1200px. The sidebar becomes modal and can be shown or hidden via the sidebar button at the top-left corner.
* Click on this button to reveal the sidebar.
* Expand the window to more than 1200px.
Observe that there is only one sidebar open.

### Additional notes

Frontend changes to made the sidebar be drawn once,
regardless of the size of the window, or any resizing actions.

Ref: issue #459
